src/System.Windows.Forms/src/System/Windows/Forms/VisualStyles/VisualStyleInformation.cs

```diff
@@ -21,9 +19,9 @@ namespace System.Windows.Forms.VisualStyles
     /// </summary>
     public static class VisualStyleInformation
     {
-        //Make this per-thread, so that different threads can safely use these methods.
+        // Make this per-thread, so that different threads can safely use these methods.
         [ThreadStatic]
-        private static VisualStyleRenderer visualStyleRenderer = null;
+        private static VisualStyleRenderer? visualStyleRenderer = null;
```

Comment:
Can we rename this while we're here?
```suggestion
        private static VisualStyleRenderer? t_visualStyleRenderer = null;
```